app/src/main/java/de/westnordost/streetcomplete/quests/camping/AddCampPower.kt

```diff
@@ -0,0 +1,39 @@
+package de.westnordost.streetcomplete.quests.camping
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.OUTDOORS
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddCampPower : OsmFilterQuestType<Boolean>() {
+
+    /* We only resurvey power_supply=yes and power_supply=no, as it might have more detailed 
+     * values from other editors, and we don't want to damage them */
+    override val elementFilter = """
+        nodes, ways with (
+          tourism=camp_site
+        )
+        and (!power_supply or (power_supply older today -4 years and power_supply ~ yes|no))
+    """
```

Comment:
```suggestion
    /* We only resurvey power_supply = yes and power_supply = no, as it might have more detailed 
     * values from other editors, and we don't want to damage them */
    override val elementFilter = """
        nodes, ways with
          tourism = camp_site and (
            !power_supply
            or power_supply older today -4 years and power_supply ~ yes|no
          )
    """
```